.concourse/kubecf.yaml

```diff
@@ -2,8 +2,16 @@
 
 s3_bucket: kubecf-ci
 s3_bucket_region: eu-central-1
+# NOTE use the following if developing
```

Comment:
If many people use the same buckets then they will end up messing each other's clusters/artifacts. I would prefer if anyone deploying a copy of the pipeline, used their own bucket. Buckets are easy to create and cheap.

Reply:
Done.
I would be happier appending commit hashes to intermediate artifacts. That feels saner, even if it's not bulletproof, since the same artifact can be built differently as pipeline definitions change.

Reply:
we do have commit hashes in the artifact names I think. As you said, 2 different pipeline might build the same commit but using different pipeline configuration, thus one overwriting the other pipelines artifacts. Separate buckets are safer imho.